.github/workflows/broken_link_checker.yml

```diff
@@ -0,0 +1,53 @@
+name: Broken link check
+
+on:
+  schedule:
+      - cron: '0 7 * * *'
+
+jobs:
+  my-broken-link-checker:
+    name: Check for broken links (${{ matrix.page }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        page:
+          - index
+          - install
+          - start
+          - tutorials
```

Comment:
Also, I think we should fix the existing broken links found by this check in this PR!

Reply:
Good point. I only found one in my testing, but forgot to push up the fix. Will do.